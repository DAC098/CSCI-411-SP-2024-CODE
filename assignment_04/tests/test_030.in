6
3 5 0
hkxkrdzjtuqpnxyk iqluubmtuoeehwjswqcrffynbkakndxqua
rkuvheltdzdyipkfqlsvzrasmnwvhoarx gdboeqahkerbirfxewkniipfdbcmbfgw
cgtcctgcggaggtgtg ggattacggctgttttgtt
xkwabwzcbgwcobkvvtynri hqwwxcxldjm
tcccccacccg tgaatgttaaa
caaaaag tatttgttatgtttt
