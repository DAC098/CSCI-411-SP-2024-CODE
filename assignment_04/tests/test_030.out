__________________hkxkrdzjtuqpnxyk
iqluubmtuoeehwjswqcrffynbkakndxqua
54
rkuvheltdzdyipkfqlsvzrasmnwvhoarx
_gdboeqahkerbirfxewkniipfdbcmbfgw
5
__cgtcctgcggaggtgtg
ggattacggctgttttgtt
6
xkwabwzcbgwcobkvvtynri
___________hqwwxcxldjm
55
tcccccacccg
tgaatgttaaa
0
________caaaaag
tatttgttatgtttt
24
